api/docs/feature-development.md

```diff
@@ -0,0 +1,238 @@
+# Feature Development in Rails
+
+## Creating a Model
```

Comment:
Not certain, but you may not need this section/can boil it down to "you should build using TDD"